doc/distrib/NodeHelpFiles/_review/DSRevitNodesUI.FilledRegionTypes.md

```diff
@@ -0,0 +1,9 @@
+## In Depth
+`Select Filled Region Type` allows for the selection of a filled region type from all types available in the active Revit document.
+
+In the example below, a new drafting view is created in the active Revit document. A filled region is constructed in the view of the "Diagonal Down" type.
```

Comment:
Rephrase last sentence for clarity : "A filled region is created in the view using "Diagonal Down" type."